std/assembly/math.ts

```diff
@@ -1249,6 +1273,109 @@ function Rf(z: f32): f32 { // Rational approximation of (asin(x)-x)/x^3
   return NativeMathf.exp(x - kln2) * scale * scale;
 }
 
+@inline /** @internal */
+function pio2_large_quot(x: f32, u: i32): i32 { // see: jdh8/metallic/blob/master/src/math/float/rem_pio2f.c
+  const pi_2_65 = reinterpret<f64>(0x3BF921FB54442D18); // 8.51530395021638647334e-20
+
+  const bits: u64[] = [
+    0xA2F9836E4E441529,
+    0xFC2757D1F534DDC0,
+    0xDB6295993C439041,
+    0xFE5163ABDEBBC561
+  ];
```

Comment:
Wondering if the array can be avoided here. If not, it should possibly be moved outside as `@lazy` so it only allocates (the array around the arraybuffer) once, once that other issue is resolved.

Reply:
Is it make sense? this function always called when sin/cos/tan used.

Reply:
Done